src/main/java/seedu/address/model/statistics/StatisticDataList.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.statistics;
+
+import static java.util.Objects.requireNonNull;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.scene.chart.PieChart.Data;
+import seedu.address.model.statistics.exceptions.DuplicateDataException;
+
+/**
+ * A list of {@code StatisticData} that enforces that no 2 StatisticData points have the same name
+ * to avoid confusing charts. As such, adding uses StatisticData#isSameName(StatisticData) in order
+ * to ensure the StatisticData being added is unique in terms of name.
```

Comment:
The 'avoid confusing chart' seems a little too informal. Does this version capture the meaning you're trying to convey?
```suggestion
 * A list of {@code StatisticData} that enforces that no two StatisticData points have the same name
 * as the charts require unique data points. As such, adding uses StatisticData#isSameName(StatisticData) in order
 * to ensure the StatisticData being added is unique in terms of name.
```

Reply:
ah yes thx for rephrasing